### PR TITLE
Installed django-smartcc

### PR DESCRIPTION
**Pull request by koalalorenzo**
Installed django-smartcc to provide better HTTP cache headers on the
website. Just for fun.

### CODE REVIEW

**Comment by valberg** [comment]
@koalalorenzo can we pin the requirement?

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 #Django==1.7.3
 https://www.djangoproject.com/download/1.8a1/tarball/
 django-environ==0.3.0
+django-smartcc
```
Comment (valberg) [inline requirements.txt]: Can we pin it to a specific version? cc/ @koalalorenzo

Comment (koalalorenzo) [inline requirements.txt]: Yep! That's better.

**Comment by emilkjer** [comment]
Nice!

**Comment by valberg** [comment]
LGTM :+1:

**Comment by emilkjer** [comment]
Deployed

**Pull request merged**